go/cmd/vtbackup/plugin_opentsdb.go

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
In order to test for vtbackup-specific metrics in an E2E test, used opentsdb plugin and write stats to file for later verification.